### PR TITLE
Adding Datasource: google_composer_user_workloads_config_map

### PR DESCRIPTION
**Pull request by spapi17**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-datasource
`google_composer_user_workloads_config_map` (beta)
```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/d/composer_user_workloads_config_map.html.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes ConfigMap in Composer Environment workloads.
```
Comment (kosteev) [inline mmv1/third_party/terraform/website/docs/d/composer_user_workloads_config_map.html.markdown:4]: I would make it consistent with changes that we did for User Workloads Secrets documentation, using wording from Composer API docs
https://github.com/GoogleCloudPlatform/magic-modules/pull/10490/files

**Review by kosteev — APPROVED** [review]

File: mmv1/third_party/terraform/website/docs/d/composer_user_workloads_config_map.html.markdown (hunk 1 above)
Comment (spapi17) [reply mmv1/third_party/terraform/website/docs/d/composer_user_workloads_config_map.html.markdown]: fixed

**Review by c2thorn — COMMENTED** [review]
The test seems to be failing with an internal error. Do you know why?

**Comment by spapi17** [comment]
Yes, this is a known transient issue with a fix in progress. The test passed locally. Could we please re-run it?

**Review by c2thorn — APPROVED** [review]

**Pull request merged**